build.sbt

```diff
@@ -3,52 +3,53 @@ lazy val micrometerVersion = "1.1.4"
 lazy val catsEffectVersion = "1.4.0"
 lazy val scalaTestVersion = "3.0.8"
 
-lazy val `http4s-micrometer-metrics` = (project in file(".")).
-  settings(
-    inThisBuild(List(
+lazy val `http4s-micrometer-metrics` = (project in file(".")).settings(
+  inThisBuild(
+    List(
       organization := "com.ovoenergy",
-      scalaVersion := "2.12.10"
-    )),
-    name := "http4s-micrometer-metrics",
-    organizationName := "OVO Energy",
-    organizationHomepage := Some(url("https://www.ovoenergy.com/")),
-    homepage := Some(url("https://github.com/ovotech/http4s-micrometer-metrics")),
-    startYear := Some(2019),
-    licenses := Seq(("Apache-2.0", url("http://www.apache.org/licenses/LICENSE-2.0"))),
-    scmInfo := Some(
-      ScmInfo(
-        url("https://github.com/ovotech/http4s-micrometer-metrics"),
-        "git@github.com:ovotech/http4s-micrometer-metrics.git"
-      )
-    ),
-    // TODO Find a way to extract those from github (sbt plugin)
-    developers := List(
-      Developer(
-        "filippo.deluca",
-        "Filippo De Luca",
-        "filippo.deluca@ovoenergy.com",
-        url("https://github.com/filosganga")
-      )
-    ),
-    scalafmtOnCompile := true,
-    libraryDependencies ++= Seq(
-      "org.typelevel" %% "cats-effect" % catsEffectVersion,
-      "org.http4s" %% "http4s-core" % http4sVersion,
-      "io.micrometer" % "micrometer-core" % micrometerVersion,
-      "org.scalatest" %% "scalatest" % scalaTestVersion % Test,
-      "org.http4s" %% "http4s-testing" % http4sVersion % Test,
-      "org.http4s" %% "http4s-server" % http4sVersion % Test,
-      "org.http4s" %% "http4s-dsl" % http4sVersion % Test,
-      "org.http4s" %% "http4s-client" % http4sVersion % Test,
-    ),
-    bintrayOrganization := Some("ovotech"),
-    bintrayRepository := "maven",
-    bintrayPackageLabels := Seq(
-      "http4s",
-      "metrics",
-      "micrometer"
-    ),
-    releaseEarlyWith := BintrayPublisher,
-    releaseEarlyNoGpg := true,
-    releaseEarlyEnableSyncToMaven := false
-  )
+      scalaVersion := "2.12.9"
```

Comment:
Why do we need to downgrade scala version?

Reply:
woops, that was accidental. Metals in vscode doesn't support .10 yet so I meant to do that just locally. I'll fix that

Reply:
fixed